app/main.py

```diff
@@ -15,5 +15,18 @@
     "Sergio": 3,
 }
 collection_of_coins = {1, 2, 25}
-
-# write your code here
+list_a = [lucky_number, pi, one_is_a_prime_number, name,
+          my_favourite_films, profile_info, marks, collection_of_coins]
+list_b = [int, float, complex, bool,
+          str, tuple, frozenset]
+copy = []
+un_copy = []
+sorted_variables = {}
+for i in range(len(list_a)):
```

Comment:
no need to use `range`
```suggestion
for elem in(list_a):
```